src/modules/displayPokemon.js

```diff
@@ -0,0 +1,21 @@
+import renderPokemon from './renderingPokemon.js';
+/* import { countLikes } from './likeCounter.js'; */
+
+export function fetchPokemonData(pokemon) {
+  const { url } = pokemon;
+  fetch(url)
+    .then((response) => response.json())
+    .then((pokeData) => {
+      renderPokemon(pokeData);
+    });
+}
+
+export function fetchPokemon() {
```

Comment:
- [ ] Great job, :clap: but I think it will be better if you can use ES6 arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html). Kindly make this for all your code